src/gui/dialogs/unit_attack.cpp

```diff
@@ -72,162 +76,8 @@ void unit_attack::pre_show()
 	keyboard_capture(&weapon_list);
 
 	// Possible TODO: If a "blank weapon" is generally useful, add it as a static member in attack_type.
```

Comment:
Does it make sense that this comment is left here?

Reply:
You moved code the comment is about, no? So either move it as well or delete it.